### PR TITLE
LPS-159343 Poshi Automation for Bug | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-159343

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase [hunk 1]
```diff
@@ -1404,6 +1404,85 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-159046. The page creator should see the tags on Asset Publisher filter selector under scoped page."
+	@priority = "3"
+	test SelectTagOnAssetPublisherUnderScopedPage {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase:1409]: FilterAssetsFromPageScopeByTag

File: portal-web/test/functional/com/liferay/portalweb/macros/asset/assetpublisher/AssetPublisherPortlet.macro [hunk 2]
```diff
@@ -603,6 +547,60 @@ definition {
 		PortletEntry.save();
 	}
 
+	macro configureScopePG {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/asset/assetpublisher/AssetPublisherPortlet.macro:550]: macro configureScope

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comments? Thanks

Hi @Tim-Cao! I've pushed new commits with the changes you had asked. As it's something small I will forward the PR. Thank!

**Comment by ozysouza** [comment]
ci:forward

**Pull request closed without merging**